_episodes/10-defensive.md

```diff
@@ -277,7 +277,9 @@ The range of each time series is represented as a pair of numbers,
 which are the time the interval started and ended.
 The output is the largest range that they all include:
 
-![Overlapping Ranges](../fig/python-overlapping-ranges.svg)
+![Four overlapping lines represent four time series, as described in the preceeding text. 
+They begin and end at different points in time, but common to all four lines is time 
+values zero through two.](../fig/python-overlapping-ranges.svg)
```

Comment:
I like the explanation you have included to describe the range overlapping, though I would modify it slightly, perhaps something close to "Four horizontal lines with a number at each end represent time series. They begin and end at different points in time, but common to all is the range 0.0 through 2.0."